aws-s3/latest/config/direct-access.md

```diff
@@ -0,0 +1,137 @@
+| connector.s3.directAccessUrl.maxExpiryTimeInSec=300 | The maximum expiry time interval that can be requested by clients - content-store specific setting. <br><br>Its value cannot exceed the system-wide configuration (`system.directAccessUrl.maxExpiryTimeInSec`) - it can only be equal or lower (otherwise DAUs for the specific content store will be disabled). <br><br>If it's not set, the default system-wide setting is used (`system.directAccessUrl.maxExpiryTimeInSec`). |
+
+> **Note:** Callers within the platform (i.e. Java interfaces) can either request a specific expiry time or rely on the default value.
+
+> **Note:** When multiple S3 buckets are used for storage in Alfresco, each S3 Content Store can be configured with either the default (common) S3 Connector-specific properties (i.e. `connector.s3.directAccessUrl.enabled` etc.), or new separate properties can be defined for each and every store (i.e. `connector.s3store1.directAccessUrl.enabled`, `connector.s3store2.directAccessUrl.enabled`, etc.).
+
+## Configuration priorities
+
+For DAUs to be usable on the service-layer, the feature must be enabled both system-wide and on the content-store(s). For the feature to be usable through REST (outside the JVM) the *rest-api configuration* must also be enabled.
+
+The `system.directAccessUrl.enabled` property is the main switch for this feature. If this is set to false, then **all** DAUs are disabled.
+
+The next configuration that controls specific DAUs is the one for the content store. The `connector.s3.directAccessUrl.enabled` property controls whether DAUs are enabled for that specific store.
+
+Whether or not a client can request a DAU by using a REST endpoint is controlled by the `restApi.directAccessUrl.enabled property`. If the REST endpoint is disabled, but the feature is enabled system-wide and on the content-store, then the DAUs will only be usable by Java clients (i.e. only service-level requests will be possible).
+
+## APIs
+
+### REST endpoints
+
+The following endpoints can be used to send requests to obtain DAUs in Content Services:
+
+* `POST /nodes/{nodeId}/request-direct-access-url`
+* `POST /nodes/{nodeId}/renditions/{renditionId}/request-direct-access-url`
+* `POST /nodes/{nodeId}/versions/{versionId}/request-direct-access-url`
+* `POST /deleted-nodes/{nodeId}/request-direct-access-url`
+* `POST /deleted-nodes/{nodeId}/renditions/{renditionId}/request-direct-access-url`
+
+Optionally, the POST body can specify an `attachment` flag. A value of `true` indicates that a download link is required; `false` indicates an embedded link is required. This defaults to `true` if it's not specified.
```

Comment:
I'd suggest removing this line. In any case, it is documented as part of the API Explorer (as well as below).